PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkBatchJobConfigurable.java

```diff
@@ -56,6 +60,11 @@ protected void createUIComponents() {
         localRunParamsPanel = new SparkLocalRunParamsPanel(getProject()).withInitialize();
         remoteConfigScrollPane = new JBScrollPane();
         setClusterSubmissionPanel(createSubmissionPanel());
+
+        this.commonPanel = new JBPanel();
+        this.commonRunParametersPanel = new SparkCommonRunParametersPanel(this.myProject, this);
+        this.commonPanel.setLayout(new BorderLayout());
+        this.commonPanel.add(commonRunParametersPanel.getComponent());
```

Comment:
Is it `commonRunParametersPanel.getComponent()` already panel? Can it be as `commonPanel`?